crates/rooch/src/commands/transaction/commands/build.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{CommandAction, TransactionOptions, WalletContextOptions};
+use async_trait::async_trait;
+use move_core_types::{
+    account_address::AccountAddress,
+    identifier::Identifier,
+    language_storage::{ModuleId, TypeTag},
+};
+use moveos_types::{move_types::FunctionId, transaction::MoveAction};
+use rooch_types::error::RoochResult;
+
+/// Get transactions by order
+#[derive(Debug, clap::Parser)]
+pub struct BuildCommand {
+    #[clap(long, required = true)]
+    pub module_address: AccountAddress,
+
+    #[clap(long, required = true)]
+    pub module_name: String,
+
+    #[clap(long, required = true)]
+    pub function_name: String,
```

Comment:
The three option can merge to one `ParsedFunctionId` like `rooch move run`

Reply:
Updated.